engines/mxnet/mxnet-engine/src/test/java/ai/djl/mxnet/engine/MxNDArrayExTest.java

```diff
@@ -0,0 +1,55 @@
+package ai.djl.mxnet.engine;
+
+import ai.djl.engine.Engine;
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDList;
+import ai.djl.ndarray.NDManager;
+import ai.djl.ndarray.internal.NDArrayEx;
+import ai.djl.ndarray.types.DataType;
+import ai.djl.ndarray.types.Shape;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class MxNDArrayExTest {
```

Comment:
We don't need this test, we have integration tests for multiBoxPrior: https://github.com/deepjavalibrary/djl/blob/master/integration/src/main/java/ai/djl/integration/tests/modality/cv/MultiBoxPriorTest.java#L30

The integration test covers both MXNet and PyTorch

Reply:
Cool - deleted.